impacket/krb5/kerberosv5.py

```diff
@@ -345,7 +349,10 @@ def getKerberosTGT(clientName, password, domain, lmhash, nthash, aesKey='', kdcH
         # probably bad password if preauth is disabled
         if preAuth is False:
             error_msg = "failed to decrypt session key: %s" % str(e)
-            raise SessionKeyDecryptionError(error_msg, asRep, cipher, key, cipherText)
+            # Commenting error below in order to return tgt, so that Kerberoast through AS-REQ can be conducted
+            # raise SessionKeyDecryptionError(error_msg, asRep, cipher, key, cipherText)
```

Comment:
`if kerbroasting:
        return tgt, None, key, None
else:
        raise SessionKeyDecryptionError(error_msg, asRep, cipher, key, cipherText)
`

Reply:
Ok so we would need to add a parameter to getKerberosTGT right?

Reply:
> Ok so we would need to add a parameter to getKerberosTGT right?

yes, as I mentioned above, _kerbroasting_ parameter should be passed to the function in getUsersSPN.py and getTGT.py

Reply:
Alright, doing the change now, but I'll name the parameter something else than `kerberoasting` since the technique is not exactly for all Kerberoastings but only for Kerberoast without pre-auth